crypto/evp/keymgmt_lib.c

```diff
@@ -92,7 +92,9 @@ void *evp_keymgmt_export_to_provider(EVP_PKEY *pk, EVP_KEYMGMT *keymgmt,
         struct import_data_st import_data;
 
         import_data.importfn =
-            want_domainparams ? keymgmt->importdomparams : keymgmt->importkey;
+            want_domainparams
```

Comment:
I think once it becomes this messy it should become a if.
Feel free to ignore if you disagree

Reply:
I disagree.